feat(Button): Alternative (secondary) modes and consistent height [GM-126] (#1413)
Codecademy/client-modules: packages/styleguide/stories/Atoms/Button/index.stories.mdx

</Canvas>

<Props story="IconButton" />

## Button Variants

> We have 2 button color modes: `light` and `dark` which are meant to be used in their respective contexts. For differentiation we also provide an alternate color scheme for lower priority buttons when used adjacently. These are `light-alt` and `dark-alt` respectively.

Reviewer: ```suggestion
We have 2 button color modes: `light` and `dark`.
For differentiation we also provide an alternate color scheme for lower priority buttons when used alongside other buttons: `light-alt` and `dark-alt`.
```

- **Don't** use a color mode outside of its context (no light buttons on navy backgrounds)
- Use `light` and `dark` modes for your primary buttons.
- Use `light-alt` and `dark-alt` modes for secondary buttons.

### Light Mode

<Canvas>
  <Story name="Light Mode">
    <ButtonScale mode="light" />
  </Story>

> </Canvas>

Reviewer 2: I think an additional code example (like 
```
<TextButton
  href=""
  mode="light-alt"
  size="normal"
>
  Click Me
</TextButton>
``` 
here would be helpful
